index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 var childProcess = require('child_process');
 var resolveFrom = require('resolve-from');
+var pify = require('pify');
+var Promise = require('pinkie-promise');
 
-module.exports = function (pth, cb) {
+module.exports = function (pth) {
 	if (typeof pth !== 'string') {
```

Comment:
Happy to change it, do you like that more? I honestly tend to prefer the "long" version these days.